gen/test_installer_aws.py

```diff
@@ -0,0 +1,10 @@
+import json
+import gen.installer.aws
+
+
+def test_gen_aws_mapping():
+    result = json.loads("{" + gen.installer.aws.gen_ami_mapping({"stable"}) + "}")
+    # check number of regions
+    assert len(result) == 10
+    # check format of response
+    assert result["ap-northeast-1"] == {'stable': 'ami-965899f7'}
```

Comment:
the copying of the AMI here is going to make it so anyone updating the AMI mappings (Which we usually do once a release). Be really nice if we programatically grabbed it